modules/registry-operator-config-resources-storage-credentials.adoc

```diff
@@ -4,10 +4,10 @@
 
 
 [id="registry-operator-config-resources-secret_{context}"]
```

Comment:
We just need to change the context to "registry-operator-config-resources-storage-credentials"

If it's easier for you, I can do merge this PR and make the change in a follow-up. Otherwise, LGTM.

Reply:
fixed, thanks for catching it